CMakeLists.txt

```diff
@@ -41,6 +41,10 @@ include(
 include(CMakeDependentOption)
 include(GNUInstallDirs)
 
+if(NOT PLATFORM)
+  set(PLATFORM "Linux")
```

Comment:
@BSolonenko is platform - some standart cmake variable? In what case it can be empty ?